validator/src/test/java/org/odftoolkit/odfvalidator/InvalidPackageTest.java

```diff
@@ -124,7 +124,30 @@ public void validateEncryptedODT() {
             t.printStackTrace(new PrintWriter(errors));
             Assert.fail(t.toString() + "\n" + errors.toString());
         }
-        Assert.assertTrue(output.contains("The document is encrypted. Validation of encrypted documents is not supported."));
+        Assert.assertTrue(output.contains("Fatal: only DEFLATED entries can have EXT descriptor"));
+        Assert.assertTrue(output.contains("Fatal: The document is encrypted. Validation of encrypted documents is not supported."));
+        java.util.logging.Logger.getLogger(getClass().getName()).log(Level.INFO, "Test result:\n{0}", output);
+    }
+
+    @Test
+    public void validateEncryptedODF13WithInvalidOpenPGP() {
+        String output = "";
+        try {
+            // password: hello
```

Comment:
Can't open the file using this password

Reply:
errr oops you're right thats copypasta from the previous test :(
there is no password, it requires a GPG key, i'm using this command to run LO and create the file, it will open without requiring a password; there's also some bug somewhere that causes LO to ask for a password when opening the file if it can't find the GPG key.

GNUPGHOME=/lo/build_icecc/workdir/CppunitTest/xmlsecurity_signing.test.user instdir/program/soffice

Reply:
but it's not really useful for this test to be able to open the document, adding GPG decryption to Validator seems definitely out of scope to me, it's enough if we can validate the manifest.xml which is never encrypted

Reply:
aha, yes, the password prompt from LibreOffice made me think it was encrypted with a password